src/pages/docs/oauth/advanced/connection-context.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: OAuth Connection Context
+description: This describes the connection context
+tags:
+  - ShipEngine Connect
+  - OAuth
+---
+
+# Connection Context
+
+Nonstandard OAuth integrations occasionally both send and expect more than an
+access token in the response and authenticated requests, respectively.
+
+To accommodate for this, MonoAuth allows a response transformer to specify
+a collection of properties in `connection_context`. Properties defined in
+`connection_context` are included in the 'code for token' (`POST /authflow/token`)
+exchange response (the
+[``](https://github.com/shipstation/monoauth/blob/main/MonoAuth/Server/Controllers/Contracts/TokenResponse.cs#L11)
+object).
+
+Values in the `connection_context` can use some _template expressions_, like
+[`{callback:`](./TEMPLATING_REF.md#callback). This allows
+an integration definition to capture values from the `redirect_uri` from the
+3rd party to include in the connection context.
+For more information on this feature see
+[PLAT-3246](https://auctane.atlassian.net/browse/PLAT-3246).
+
+The name `connection_context` comes from the ShipEngine Connect Platform and
+iPaaS as a whole. `connection_context` may include data entered by the user
+from the data-driven connection UI described by an integration and/or data
```

Comment:
link to Forms https://connect-git-update-oauth-shipengine.vercel.app/docs/v2/structure#forms and get rid of data-driven